buildSrc/version.properties

```diff
@@ -60,9 +60,9 @@ bouncycastle=1.78
 randomizedrunner  = 2.7.1
 junit             = 4.13.2
 hamcrest          = 2.1
-mockito           = 5.12.0
+mockito           = 5.14.1
 objenesis         = 3.2
-bytebuddy         = 1.14.9
+bytebuddy         = 1.15.4
```

Comment:
@reta Can we backport this version update change to 2.x?

Reply:
Let me provide some context here. I'm working on the [skills repo](https://github.com/opensearch-project/skills). I've created a [PR](https://github.com/opensearch-project/skills/pull/466) to fix the CI error by bumping the version of `bytebuddy`, but it is very strange that its [backport-2.x PR](https://github.com/opensearch-project/skills/pull/467) meets with the version conflict error:

```
* What went wrong:
Execution failed for task ':compileTestJava'.
> Could not resolve all dependencies for configuration ':testCompileClasspath'.
   > Conflict found for the following module:
       - net.bytebuddy:byte-buddy between versions 1.15.10 and 1.15.4
```

Do you have any hint on why? I do not have much context on these dependencies.

Reply:
The 2.x is on 15.10 as well (https://github.com/opensearch-project/OpenSearch/pull/16657), I will take a look shortly why the CI for the [skills repo](https://github.com/opensearch-project/skills) repo fails.